value-processor/src/org/immutables/value/processor/Immutables.generator

```diff
@@ -2479,6 +2497,25 @@ this.[n] = [valueFromValue v][invokeSuper v].[v.names.get]()[/valueFromValue];
 [/for]
 [/template]
 
+[template public buildFromMandatoryAttribute Invokable buildAttribute Attribute v String varName]
+[if v.isMandatory]
+if ([varName].[v.names.isSet]()) {
+  [buildAttribute v]
+}
+[else]
+  [buildAttribute v]
+[/if]
+[/template]
+
+[template public dynamicFromModifiableCheck Type type String varName String returnValue]
+[if type.kind.isModifiable]
+if ([varName] instanceof [type.typeModifiable.relativeRaw]) {
```

Comment:
maybe as a matter of style/our preference we should add `[type.generics.unknown]` after `.relativeRaw]` so if type have generic parameters, it will output trailing `<?>` after raw type name (we try to avoid any use or raw types, even in instanceof I prefer to check it agains generic type with unknown arguments).
This also prompts to have at least one compilation test for value type having generic type parameters. We need check that we don't have new unsuppressed unchecked cast warnings in generated code.